packages/gatsby-source-wikipedia/src/fetch.js

```diff
@@ -70,16 +70,16 @@ const search = ({ query, limit }) =>
         return {
           title,
           description: descriptions[i],
-          id: /en.wikipedia.org\/wiki\/(.+)$/.exec(urls[i])[1],
+          id: term,
         }
       })
     )
 
-const getArticle = name =>
-  fetch(`https://en.m.wikipedia.org/wiki/${name}?action=render`)
+const getArticle = name => lang =>
```

Comment:
There is no reason to use currying here. Just use two args:

```suggestion
const getArticle = (name, lang) =>
```